addOns/client/src/main/java/org/zaproxy/addon/client/ExtensionClientIntegration.java

```diff
@@ -368,6 +369,9 @@ public void postInit() {
                 LOGGER.error(
                         "Failed to get or create Firefox profile {}", ZAP_FIREFOX_PROFILE_NAME);
             }
+        } catch (WebDriverException e) {
+            // Will happen if Firefox is not available.
+            LOGGER.debug(e.getMessage(), e);
```

Comment:
Don't we want users, logs, and the auth report to contain this if it's the case?

Is this based on: https://groups.google.com/g/zaproxy-users/c/cAgxv-X57lo/m/BZFYTAKfIAAJ ?